src/mbed_os_tools/test/host_tests_plugins/host_test_plugins.py

```diff
@@ -221,16 +221,18 @@ def check_parameters(self, capability, *args, **kwargs):
             return False
         return True
 
-    def run_command(self, cmd, shell=True):
+    def run_command(self, cmd, shell=True, stdin = None, stdout = None):
         """! Runs command from command line.
         @param cmd Command to execute
         @param shell True if shell command should be executed (eg. ls, ps)
+        @param stdin A custom stdin for the process running the command (defaults to None)
+        @param stdout A custom stdout for the process running the command (defaults to None)
```

Comment:
Doesn't look like we ever use `stdout`. I realize it pairs nicely with `stdin`, but since we don't have any use for it, would you mind removing this argument (and its other mentions in the rest of this PR)?

Reply:
Done